python/lsst/ts/ess/mock/mock_temperature_sensor.py

```diff
@@ -0,0 +1,61 @@
+# This file is part of ts_Dome.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["MockTemperatureSensor"]
+
+import logging
+
+from lsst.ts import salobj
+from lsst.ts.ess.mock.ess_temperature_4ch_c import EssTemperature4ChC
+
+
+class MockTemperatureSensor:
+    """Mock Temperature Sensor.
+    """
+
+    def __init__(self):
+        self.myData = EssTemperature4ChC()
+        # Time keeping
+        self.current_tai = 0
+        self.previous_tai = 0
+        self.log = logging.getLogger("MockTemperatureSensor")
+        self.log.info("__init__")
+
+    async def readInstrument(self):
+        self.log.info("readInstrument")
+        # Mock the behavior that the temperature data are only updated by the
+        # real sensors every 1 second.
+        await self.determine_current_tai()
+        if self.current_tai - self.previous_tai > 1.0:
+            self.myData.timestamp = self.current_tai
+            self.myData.TemperatureC01 = 20.0
```

Comment:
I would rather implement 20.0 + random number (such as random.gauss(20.0, 3)?). Same for following channels. This makes EUI/GUIs more happy displaying change in values.

Reply:
Great suggestion. Done.